prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -466,10 +476,8 @@ def prompt_processor() -> Any:
                     Output just the JSON string. No explanation is required. \
                     If you cannot extract the JSON string, output {{}}"
                 answer = run_completion(
-                    llm_helper=llm_helper,
-                    llm_li=llm_li,
-                    prompt=prompt,
-                    adapter_instance_id=adapter_instance_id,
+                    llm,
+                    prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
Done